CIME/Tools/standard_script_setup.py

```diff
@@ -15,22 +15,16 @@ def check_minimum_python_version(major, minor):
     >>> check_minimum_python_version(sys.version_info[0], sys.version_info[1])
     >>>
     """
-    msg = (
-        "Python "
-        + str(major)
-        + ", minor version "
-        + str(minor)
-        + " is required to run CIME. You have "
-        + str(sys.version_info[0])
-        + "."
-        + str(sys.version_info[1])
-    )
-    assert sys.version_info[0] > major or (
-        sys.version_info[0] == major and sys.version_info[1] >= minor
-    ), msg
-
-
-check_minimum_python_version(3, 8)
+    if sys.version_info[0] < major or (
+        sys.version_info[0] == major and sys.version_info[1] <= minor
+    ):
+        print(
+            f"You're running an out-of-date python version {sys.version_info.major}.{sys.version_info.minor}.\nCIME may not work as expected, consider upgrading to python > {major}.{minor}.",
```

Comment:
I don't like the wording here.   It will tell you "CIME may not work as expected, consider upgrading to python > 3.6."   and I submit that that's too old a version to recommend upgrading to.   How about "CIME may not work as expected, consider upgrading to a supported python version as documented in https://devguide.python.org/versions/"